src/context.hh

```diff
@@ -22,7 +22,7 @@ class AliasRegistry;
 struct JumpList
 {
     void push(SelectionList jump, Optional<size_t> index = {});
-    const SelectionList& forward(Context& context, int count);
+    const SelectionList& forward(const Context& context, int count);
```

Comment:
Here, its surprising that forward takes a const Context but backward not. I think thats due to backward potentially mutating the jump list, which forward never does, but it still feels curious to put that in the function contract (as a caller, its strange that if you have a const Context, you can jump forward but not backwards)